src/Paraunit/Parser/ParserCompilerPass.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Paraunit\Parser;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * Class ParserCompilerPass
+ */
+class ParserCompilerPass implements CompilerPassInterface
+{
+
+    const TAGGED_SERVICES = 'log_parser';
+    const TO_COMPILE      = 'paraunit.parser.json_log_parser';
+
+    /**
+     * @inheritdoc
+     */
+    public function process(ContainerBuilder $container)
+    {
+
+        $taggedServices = $container->findTaggedServiceIds(self::TAGGED_SERVICES);
+
+        uasort(
+            $taggedServices,
+            function ($a, $b) {
```

Comment:
Minor WS issue: https://scrutinizer-ci.com/g/facile-it/paraunit/inspections/e5c5a0ff-09c7-4330-9f85-dcf9dc969e67/patches?status=new